src/test/java/com/easyrestaurant/tests/CreateAdministratorProfile.java

```diff
@@ -0,0 +1,98 @@
+package com.easyrestaurant.tests;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+
+import java.time.Duration;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.Test;
+
+public class CreateAdministratorProfile {
+	
+	WebDriver driver;
+	static LoadProperties properties = new LoadProperties();
+	private final static String administratorName = "Zaio Baio",
+			administratorEmail = "zaio@bg.eu",
+			securePass = "Neznamtaziparola2",
+			phoneNumber = "0872142767";
+
+	@Test(description = "Create Administrator Profile From owner's menu",
+			priority = 1)
+	public void createWaiterProfile() throws InterruptedException {
+		//Load username and password from config.properties
+		properties.loadProperties();
+		final String email = properties.userEmail, password = properties.userPassword;
+			
+		System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver");
+		driver = new ChromeDriver();
+		driver.manage().window().maximize();
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+		
+		String url = "http://localhost:3000";
+		driver.get(url);
+		
+		WebElement login = driver.findElement(By.xpath("//*[@id=\"root\"]/header/div/div/div/a[1]"));
+		login.click();
+		WebElement fieldEmail = driver.findElement(By.xpath("//*[@id=\"root\"]/main/div/div[2]/form/div/div[1]/div/div/input"));
+		fieldEmail.sendKeys(email);
+		WebElement fieldPass = driver.findElement(By.xpath("//*[@id=\"root\"]/main/div/div[2]/form/div/div[2]/div/div/input"));
+		fieldPass.sendKeys(password);
+		WebElement logIn = driver.findElement(By.xpath("//*[@id=\"root\"]/main/div/div[2]/form/div/div[3]/div/button"));
+		logIn.click();
+		WebElement profile = driver.findElement(By.xpath("//html//div[@id='root']/header/div//button[@type='button']//div[.='J']"));
+		profile.click();
+		WebElement profileLink = driver.findElement(By.xpath("//html/body/div[2]/div[2]/ul/a"));
+		profileLink.click();
+		WebElement myRestaurantsMenu = driver.findElement(By.xpath("//html/body/div/main/div/div/div/div[2]/div/a[4]"));
+		myRestaurantsMenu.click();
+		
+		//I need to create restaurant before launch next step!
+		WebElement showOptions = driver.findElement(By.xpath("//html/body/div/main/div/div/div/div[1]/div[1]/div/div/div[2]/div/div[1]/button"));
```

Comment:
When I ran the test, I had this error message and test didn't pass: "Unable to locate element: {"method":"xpath","selector":"//html/body/div/main/div/div/div/div[1]/div[1]/div/div/div[2]/div/div[1]/button"}"

Reply:
Read the comment on code //I need to create restaurant before launch next step!
You need to create a restorant manualy in order to finish the test. After the restorant is created you could run both test "create admin profile" and "create waiter profile". Also the restorant should be save in your DB, so no need to recreate it every time.